sunpy/visualization/animator/wcs.py

```diff
@@ -110,7 +103,7 @@ def _get_wcs_labels(self):
         Read first the axes names property of the wcs and fall back to physical types.
         """
         # world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.
-        world_axis_names = getattr(self.wcs, "world_axis_names", [''] * self.wcs.world_n_dim)
+        world_axis_names = self.wcs.world_axis_names
         # Return the name if it is set, or the physical type if it is not.
         return [l or t for l, t in zip(world_axis_names, self.wcs.world_axis_physical_types)]
```

Comment:
Might as well drop the intermediate variable as well I think:
```suggestion
        world_axis_names = self.wcs.world_axis_names
        # Return the name if it is set, or the physical type if it is not.
        return [l or t for l, t in zip(self.wcs.world_axis_names, self.wcs.world_axis_physical_types)]
```

Reply:
Then delete the first comment too?

Reply:
`world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.`

Reply:
Yep, deleting the comment seems like the right thing to do